app/build.gradle

```diff
@@ -27,8 +27,8 @@ dependencies {
   implementation 'androidx.appcompat:appcompat:1.0.2'
   implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
   implementation "androidx.cardview:cardview:1.0.0"
-  implementation 'com.android.support:appcompat-v7:28.0.0'
-  implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+  implementation 'androidx.appcompat:appcompat:1.0.0'
+  implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
AndroidX libraries are directly above this I think we can remove this duplication.

Reply:
removed